backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/RTTIGenerator.kt

```diff
@@ -195,11 +195,13 @@ internal class RTTIGenerator(override val context: Context) : ContextUtils {
 
         val instanceSize = getInstanceSize(bodyType, className)
 
-        val superTypeOrAny = irClass.getSuperClassNotAny() ?: context.ir.symbols.any.owner
+        val superTypeOrAny = irClass.getSuperClassNotAny().let { superType ->
+            if (superType == null || !superType.requiresRtti()) context.ir.symbols.any.owner else superType
```

Comment:
What is this required for?
It would be cleaner to avoid generating `irClass` too, and leave this code crashing if `irClass.requiresRtti()`, but one of its super types doesn't.

Reply:
For now, It is required for Kotlin subclasses of Objective-C classes. I'll try to eliminate RTTI for them, and if it work out, fix this code.